pkgs/applications/blockchains/btcpayserver/default.nix

```diff
@@ -15,13 +15,13 @@ in
 
 stdenv.mkDerivation rec {
   pname = "btcpayserver";
-  version = "1.0.7.0";
```

Comment:
The deleted hunk must be `version = "1.0.7.2";`, otherwise the patch doesn't apply to master.

Reply:
My bad, forgot to fast-forward master